code/modules/client/preference_setup/loadout/items/accessories.dm

```diff
@@ -703,3 +703,9 @@
 	display_name = "visegradi patterned sweater"
 	path = /obj/item/clothing/accessory/sweater/visegradi
 	flags = GEAR_HAS_DESC_SELECTION | GEAR_HAS_COLOR_SELECTION
+
+/datum/gear/accessory/nco_badge
+	display_name = "imperial army badge"
+	path = /obj/item/clothing/accessory/dominia/nco_badge
+	flags = GEAR_HAS_DESC_SELECTION
+	culture_restriction = list(/singleton/origin_item/culture/dominia, /singleton/origin_item/culture/dominian_unathi)
```

Comment:
item's desc
> A sleeve patch worn by non-commissioned officers of the Imperial Army.

I really do not like the idea of adding actual military items to the loadout.

General military items should be fine, stuff like the TCFL jackets, or gadpathur things. But not actual military badges, that show the character holding some rank or whatever.

Horizon crew is working for SCC or the other corps. They should not be holding ranks, as if they were still in active duty, and should not be parading around a corporate ship wearing badges or insignia with these ranks.

Reply:
What change is it you want here, sorry?

I don't disagree regarding the patch - was asked to put it in so here I am. Happy to remove it + the medical cape. Not following what you need for the other one.

Reply:
I just wanted the NCO patch removed from the PR, yeah